lib/renderer/override.js

```diff
@@ -4,5 +4,6 @@ const {ipcRenderer} = require('electron')
 
 const {guestInstanceId, openerId} = process
 const hiddenPage = process.argv.includes('--hidden-page')
+const usesNativeWindowOpen = process.argv.includes('--native-window-open')
```

Comment:
It would be great to support this as well when `contextIsolation` is enabled, which mean this variable would need to be threaded through in `lib/isolated_renderer/init.js`.